.gitattributes

```diff
@@ -1 +1,18 @@
-bench/scalajs-runtime-sourcemap.js binary
\ No newline at end of file
+dist/source-map.debug.js binary
```

Comment:
It makes `git diff` not spew changes for that file as well. I think it is worth it since the best way to handle conflicts in the generated files is to kill them and regenerate them.